### PR TITLE
Test transformations

### PR DESCRIPTION
**Pull request by Michal-Suhak**

### CODE REVIEW

File: src/quokka_editor_back/utils/ot.py [hunk 1]
```diff
@@ -100,7 +91,5 @@ def apply_operation(document_content: list[str], op: OperationSchema) -> list[st
             combined = [before + middle[0] + after]
     elif op.type == OperationType.DELETE:
         combined = [before + middle[0] + after]
-    else:
-        raise Exception("Invalid Operation!!!!")
```
Comment (DamianCzajkowski) [inline src/quokka_editor_back/utils/ot.py:104]: Why this is removed?

File: src/quokka_editor_back/utils/ot.py [hunk 2]
```diff
@@ -23,6 +23,7 @@ def adjust_position(
             line=new_pos.line,
             ch=new_pos.ch + len(prev_text),
         )
+    # There is no chance to achieve this
```
Comment (DamianCzajkowski) [inline src/quokka_editor_back/utils/ot.py:26]: Instead of adding comment, remove this las return

File: src/quokka_editor_back/utils/ot.py (hunk 1 above)
Comment (DamianCzajkowski) [reply src/quokka_editor_back/utils/ot.py:104]: Maybe add custom exception here

File: src/quokka_editor_back/tests/utils/test_ot.py [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+        text=["text"],
+        type=OperationType.DELETE,
+        revision=1,
+    )
+
+    # When
+    result = transform(new_op, prev_op)
+
+    # Then
+    # Need explanation
```
Comment (DamianCzajkowski) [inline src/quokka_editor_back/tests/utils/test_ot.py:242]: Write a question here what needs to be explained

Comment (Michal-Suhak) [reply src/quokka_editor_back/tests/utils/test_ot.py:242]: It is outdated now. You explained it to me on Sunday

File: src/quokka_editor_back/utils/ot.py (hunk 1 above)
Comment (Michal-Suhak) [reply src/quokka_editor_back/utils/ot.py:104]: Here, an exception is not required. The Pydantic schema will validate if the operation in the body is valid

**Review by DamianCzajkowski — APPROVED** [review]

**Pull request merged**